Python/cp/max_sum_k_consecutive_elements.py

```diff
@@ -0,0 +1,33 @@
+def maxSumofKelements(arr, n, k):
```

Comment:
Add description of the problem in commented form.

Reply:
done